waspc/test/Generator/FileDraft/JsonFileDraftTest.hs

```diff
@@ -0,0 +1,41 @@
+{-# LANGUAGE DeriveAnyClass #-}
```

Comment:
Since you introduced BytesFileDraft, and createJsonFileDraft is just a helper function for creating BytesFileDraft from JSON, I think we should instead have BytesFileDraftTest test file as a basic thing, and then possibly a separate small test for that createJsonFileDraft function, just to check it correctly creates BytesFileDraft.

Reply:
Since the only way to create a bytes file draft right now is to use JSON I've kep the test as is but renamed it to be a general BytesFileDraftTest, as that makes sense.

If we get more things that create BytesFiles they can be added here, or in their own test files.